tests/sentry/web/api/tests.py

```diff
@@ -105,3 +105,27 @@ def test_superuser(self):
         assert data["user"]
         assert data["user"]["email"] == user.email
         assert data["user"]["isSuperuser"]
+
+    def test_url_prefix_unauthenticated(self):
```

Comment:
Should we have a test for when the user is not in any organization, or the request is to an account settings page?